CMSIS/NN/Tests/UnitTest/TestCases/test_arm_depthwise_conv_3x3_s8/Unity/unity_test_arm_depthwise_conv_3x3_s8.c

```diff
@@ -16,42 +16,29 @@
  * limitations under the License.
  */
 
+#include <stdbool.h>
+#include <stdint.h>
 #include <stdio.h>
-#include <string.h>
 #include <stdlib.h>
-#include <stdint.h>
-#include <stdbool.h>
+#include <string.h>
 
-#include "unity.h"
 #include "../test_arm_depthwise_conv_3x3_s8.c"
+#include "unity.h"
 
 /* This function is called from the autogenerated file.
  * The name must be exactly like this
  */
 void setUp(void)
-{
-  /* This is run before EACH TEST */
+{ /* This is run before EACH TEST */
 }
 
 /* This function is called from the autogenerated file.
  * The name must be exactly like this
  */
-void tearDown(void)
-{
-
-}
+void tearDown(void) {}
 
-void test_depthwise_kernel_3x3_arm_depthwise_conv_3x3_s8(void)
-{
-  depthwise_kernel_3x3_arm_depthwise_conv_3x3_s8();
-}
+void test_depthwise_kernel_3x3_arm_depthwise_conv_3x3_s8(void) { depthwise_kernel_3x3_arm_depthwise_conv_3x3_s8(); }
 
-void test_depthwise_kernel_3x3_arm_depthwise_conv_3x3_1_s8(void)
-{
-  depthwise_kernel_3x3_arm_depthwise_conv_3x3_1_s8();
-}
+void test_depthwise_kernel_3x3_arm_depthwise_conv_3x3_1_s8(void) { depthwise_kernel_3x3_arm_depthwise_conv_3x3_1_s8(); }
```

Comment:
The format here doesn't match that of the other one liner functions.. could you recheck on this?